R/utilities-parameter.R

```diff
@@ -30,9 +29,10 @@ getParameters <- function(path, container) {
 #'
 #' @return [Parameter] The parameter with the given path or null if not found
 #' @examples
-#'\dontrun{
-#' getParameter(c("Organism", "Liver", "Volume"), sim))
-#' }
+#'
+#' simPath <- system.file("extdata", "simple.pkml", package = "ospsuite")
+#' sim <- loadSimulation(simPath)
+#' param <- getParameter(c("Organism", "Liver", "Volume"), sim)

```

Comment:
Pretty neat that those examples are being run in fact. You are sure that the example works as expected